chapters/ch19.md

```diff
@@ -1,28 +1,27 @@
 # 19. Takeaways: Basic error analysis
 ->
-
+# 19. Điều cần nhớ: Phân tích lỗi cơ bản
 
 * When you start a new project, especially if it is in an area in which you are not an expert, it is hard to correctly guess the most promising directions.
 ->
-
+* Khi bạn bắt đầu một dự án mới, đặc biệt nếu đó là một lĩnh vực mà bạn không phải là chuyên gia, nó rất khó để dự đoán được những hướng giải quyết triển vọng nhất.
 
 * So don’t start off trying to design and build the perfect system. Instead build and train a basic system as quickly as possible—perhaps in a few days. Then use error analysis to help you identify the most promising directions and iteratively improve your algorithm from there.
 ->
-
+* Vì vậy đừng bắt đầu với việc thiết kế và xậy dựng một hệ thống hoàn hảo. Thay vào đó hãy xây dựng và huấn luyện một hệ thống cơ bản một cách nhanh nhất có thể-thậm chí chỉ trong một vài ngày. Sau đó, sử dụng phân tích lỗi để giúp bạn xác định những hướng triển vọng và lặp đi lặp lại cải thiện thuật toán của bạn từ đó.
```

Comment:
* "đừng **cố** bắt đầu ...", từ "cố" ở đây quan trọng. Nếu không có từ này, câu này dễ bị hiểu sai thành không nên xây dựng hệ thống hoàn hảo.

* đề xuất:
> Sau đó, sử dụng phân tích lỗi để giúp bạn xác định những hướng _đi_ triển vọng và lặp đi lặp lại _việc_ cải thiện thuật toán của bạn từ _hệ thống_ đó.

Reply:
"Lặp đi lặp lại cải thiện thuật toán" -> "Lặp đi lặp lại **việc** cải thiện..."
Bạn thiếu chữ "việc" rồi